src/orders/components/OrderTransactionCardTitle/styles.ts

```diff
@@ -0,0 +1,57 @@
+import { makeStyles } from "@saleor/macaw-ui";
+
+export const useStyles = makeStyles(
+  theme => ({
+    title: {
+      width: "100%",
+      display: "flex",
+      justifyContent: "space-between",
+      alignItems: "center",
+    },
+    methodName: {
+      display: "flex",
+      gap: theme.spacing(1),
+      alignItems: "center",
+      fontWeight: 600,
+    },
+    dataDisplay: {
+      display: "flex",
+      gap: theme.spacing(2),
+
+      "& > dl": {
+        minWidth: "73px", // aligns amounts with < 10 to each other
+      },
+    },
+  }),
+  { name: "OrderTransactionCardTitle" },
+);
+
+export const useMoneyDisplayStyles = makeStyles(
+  theme => ({
+    wrapper: {
+      color: theme.palette.saleor.main[2],
+      margin: 0,
+      textAlign: "left",
+    },
+    label: {
+      fontWeight: 600,
+      lineHeight: "12px",
+      fontSize: theme.spacing(1.25),
+      textTransform: "uppercase",
+    },
+    moneyWrapper: {
+      margin: 0,
+      fontSize: theme.spacing(1.6875),
+      lineHeight: "21.6px",
```

Comment:
This looks very bad, where do these values come from?

Reply:
Those values came from Figma ;)

Reply:
In my opinion, we do not have to follow design 1:1, but we need to understand the assumptions the design has and follow them.  So, I think, as long as we follow design assumptions, we can safely round that type of very exact value or replace it with some variable for simplicity, so it will make code easier to comprehend and easier to write. There should be rare cases in which we should force ourselves to be that precise. I would argue with any designer who disagree with it.

Reply:
sure enough, I've changed the font-size to look good without those weird values